packages/overlay/src/App.tsx

```diff
@@ -59,10 +59,18 @@ export default function App({
       setOpen(false);
     };
 
+    const handleKeyPress = (event: KeyboardEvent) => {
+      if (event && event.ctrlKey && event.shiftKey && event.key.toLowerCase() === 's') {
+        setOpen(prevValue => !prevValue);
+      }
+    };
+
+    document.addEventListener('keydown', handleKeyPress);
```

Comment:
Is there a specific reason you put this into the `useEffect` block?

Asking because if we can move it to the top level of the `App` component, I'd prefer using the `useKeyPress` hook we already use for the ESC key navigation.